resources/js/photoinit.js

```diff
@@ -196,6 +209,32 @@ function initPhotoSwipeFromDOM (gallerySelector) {
         photoBooth.toggleMailDialog(img);
     });
 
+    /* slideshow management */
+    $(ssButtonClass).on('click touchstart', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        // toggle slideshow on/off
+        setSlideshowState(this, !ssRunning);
+    });
+
+    function setSlideshowState(el, running) {
+        if (running) {
+            setTimeout(gotoNextSlide, ssDelay / 2.0);

```

Comment:
Why do we go to the next slide after half the time?

Reply:
https://github.com/dimsemenov/PhotoSwipe/issues/753#issuecomment-225374602